utils/annotations.py

```diff
@@ -10,8 +10,43 @@
 
 
 class Annotations:
-    def __init__(self, annotation_store):
-        self.annotation_store = annotation_store
+    def __init__(self, annotation_store, global_store):
+
+        if annotation_store:
+            slices = []
+            for annotation_class in annotation_store:
+                slices.extend(list(annotation_class["annotations"].keys()))
+            slices = set(slices)
+            annotations = {key: [] for key in slices}
+
+            for annotation_class in annotation_store:
+
+                for image_idx, slice_data in annotation_class["annotations"].items():
+                    for shape in slice_data:
+                        self._set_annotation_type(shape)
+                        self._set_annotation_svg(shape)
+                        # the id of the annotation is the concatenation of its rgb color
+                        id = int(
+                            annotation_class["color"]
+                            .replace("rgb(", "")
+                            .replace(")", "")
+                            .replace(", ", "")
```

Comment:
another idea: with the updated color logic, I generate a UUID for each class (this is what MATCH is used with now). It is saved in the annotation_class_store as class_id. Maybe best to use this actually to remain consistent.

Reply:
I updated things to use incrementing ints instead of uuids, which is helpful because when transforming the annotations to a pixel mask I need a value to assign to the pixels for each class. uuid doesn't work for this since it's a string, and I'd rather keep things simple instead of creating a new int value on top of the uuid.

Reply:
Ok perfect!